src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/RecordedAct1CompletionPaymentFakeEvent.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events
+{
+    public class RecordedAct1CompletionPaymentFakeEvent
+    {
+        public DateTimeOffset EventTime { get; set; }
+        public long? ApprenticeshipId { get; set; }
```

Comment:
Why is this null, anyway? Surely it could never be? (see above comment about the HasValue)

Reply:
It's Null in Payments (as it belongs to the a base class in payments where it is optional). Therefore it has be a nullable type in the message

Reply:
:+1: